README.md

```diff
@@ -1,2 +1,457 @@
-# unione-php
-UniOne Email API PHP Library
+# UniOne PHP client
+
+[![Latest Stable Version](http://poser.pugx.org/unione/unione-php/v)](https://packagist.org/packages/unione/unione-php)
+[![Total Downloads](http://poser.pugx.org/unione/unione-php/downloads)](https://packagist.org/packages/unione/unione-php)
+
+This is the UniOne PHP SDK. This SDK contains methods for easily interacting
+with the UniOne API. For additional
```

Comment:
I would remove the first and last sentences, so the final result is:

`This SDK contains methods for easily interacting with the UniOne API: https://docs.unione.io/en/web-api-ref#web-api.`